admin/menu/class-replacevar-editor.php

```diff
@@ -24,17 +24,24 @@ class WPSEO_Replacevar_Editor {
 	 */
 	private $description;
 
+	/**
+	 * @var string The post type for context.
+	 */
+	private $post_type;
```

Comment:
Ah no. This also can be an archive or a taxonomy/term. `page_type` might be a better name.